### DISCUSSION TITLE
Official Discord / Slack Channel?

### DISCUSSION THREAD

**Discussion by rakshit087** [hoppscotch/hoppscotch#1998] [Ideas]
Can we work on an official Discord or Slack channel? It would make collaboration easier and communication faster.
Also beginners (Like Me) who want to contribute would be able to get some guidance.   
Edit: Sorry I missed the badge which was right below the image.

**Comment by AndrewBastin** [comment]
@rakshit087 we already do have a Discord server for Hoppscotch. Please check out the link in the README. Do you have anything specific in mind ?

**Reply by rakshit087** [reply]
I am sorry I checked Readme but unable to find the link. 
I will recheck, Thanks 👍

**Reply by liyasthomas** [reply]
Here's a link to "Support and Contact" section: https://github.com/hoppscotch/hoppscotch#support
